### ISSUE TITLE
Crash in IntelliJ IDEA Scio Plugin

### ISSUE THREAD

**Issue by mikaelstaldal** [spotify/scio#966]
I got this error when opening a Scio project in IntelliJ IDEA.

Scio plugin 0.1.9.

IntelliJ config:
```
2017-12-07 15:37:56,637 [     13]   INFO -        #com.intellij.idea.Main - IDE: IntelliJ IDEA (build #IU-173.3727.127, 27 Nov 2017 14:56) 
2017-12-07 15:37:56,637 [     13]   INFO -        #com.intellij.idea.Main - OS: Linux (4.10.0-40-generic, amd64) 
2017-12-07 15:37:56,637 [     13]   INFO -        #com.intellij.idea.Main - JRE: 1.8.0_144-b01 (Oracle Corporation) 
2017-12-07 15:37:56,637 [     13]   INFO -        #com.intellij.idea.Main - JVM: 25.144-b01 (Java HotSpot(TM) 64-Bit Server VM) 
2017-12-07 15:37:56,638 [     14]   INFO -        #com.intellij.idea.Main - JVM Args: -Xbootclasspath/a:/opt/idea-IU-173.3727.127/lib/boot.jar -Xms128m -Xmx750m -XX:ReservedCodeCacheSize=240m -XX:+UseConcMarkSweepGC -XX:SoftRefLRUPolicyMSPerMB=50 -ea -Dsun.io.useCanonCaches=false -Djava.net.preferIPv4Stack=true -XX:+HeapDumpOnOutOfMemoryError -XX:-OmitStackTraceInFastThrow -Dawt.useSystemAAFontSettings=lcd -Dsun.java2d.renderer=sun.java2d.marlin.MarlinRenderingEngine -XX:ErrorFile=/home/mikaelstaldal/java_error_in_IDEA_%p.log -XX:HeapDumpPath=/home/mikaelstaldal/java_error_in_IDEA.hprof -Didea.paths.selector=IntelliJIdea2017.3 -Djb.vmOptionsFile=/home/mikaelstaldal/.IntelliJIdea2017.3/config/idea64.vmoptions -Didea.jre.check=true 
2017-12-07 15:37:56,639 [     15]   INFO -        #com.intellij.idea.Main - ForkJoinPool.commonPool: java.util.concurrent.ForkJoinPool@4690f34a[Running, parallelism = 3, size = 0, active = 0, running = 0, steals = 0, tasks = 0, submissions = 0] 
2017-12-07 15:37:56,639 [     15]   INFO -        #com.intellij.idea.Main - ext: /opt/jvm/jdk1.8.0_144/jre/lib/ext: [localedata.jar, jfxrt.jar, sunpkcs11.jar, sunec.jar, jaccess.jar, dnsns.jar, cldrdata.jar, meta-index, sunjce_provider.jar, zipfs.jar, nashorn.jar] 
2017-12-07 15:37:56,639 [     15]   INFO -        #com.intellij.idea.Main - JNU charset: UTF-8 
2017-12-07 15:37:56,648 [     24]   INFO -        #com.intellij.idea.Main - JNA library (64-bit) loaded in 9 ms 
```
Stack trace:
```
scala.Predef$.refArrayOps([Ljava/lang/Object;)Lscala/collection/mutable/ArrayOps;
java.lang.NoSuchMethodError: scala.Predef$.refArrayOps([Ljava/lang/Object;)Lscala/collection/mutable/ArrayOps;
	at com.spotify.scio.ScioInjector$$anonfun$getConstructorProps$2.apply(ScioInjector.scala:192)
	at com.spotify.scio.ScioInjector$$anonfun$getConstructorProps$2.apply(ScioInjector.scala:193)
	at scala.Option.map(Option.scala:146)
	at com.spotify.scio.ScioInjector.getConstructorProps(ScioInjector.scala:191)
	at com.spotify.scio.ScioInjector.getTupledMethod(ScioInjector.scala:210)
	at com.spotify.scio.ScioInjector$$anonfun$injectInners$1.apply(ScioInjector.scala:110)
	at com.spotify.scio.ScioInjector$$anonfun$injectInners$1.apply(ScioInjector.scala:106)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
	at scala.collection.immutable.List.flatMap(List.scala:352)
	at com.spotify.scio.ScioInjector.injectInners(ScioInjector.scala:106)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.SyntheticMembersInjector$.$anonfun$injectInners$1(SyntheticMembersInjector.scala:132)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.SyntheticMembersInjector$.$anonfun$injectInners$1$adapted(SyntheticMembersInjector.scala:131)
	at scala.collection.immutable.HashSet$HashSet1.foreach(HashSet.scala:320)
	at scala.collection.immutable.HashSet$HashTrieSet.foreach(HashSet.scala:976)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.SyntheticMembersInjector$.injectInners(SyntheticMembersInjector.scala:131)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTypeDefinition.syntheticTypeDefinitionsImpl(ScTypeDefinition.scala:77)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTypeDefinition.syntheticTypeDefinitionsImpl$(ScTypeDefinition.scala:77)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.ScTypeDefinitionImpl.syntheticTypeDefinitionsImpl(ScTypeDefinitionImpl.scala:49)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.syntheticTypeDefinitions(ScTemplateDefinition.scala:183)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.syntheticTypeDefinitions$(ScTemplateDefinition.scala:182)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.ScTypeDefinitionImpl.syntheticTypeDefinitions(ScTypeDefinitionImpl.scala:49)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$TypeNodes$.processScala(TypeDefinitionMembers.scala:282)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.MixinNodes.build(MixinNodes.scala:336)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.MixinNodes.build(MixinNodes.scala:308)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.inner$2(TypeDefinitionMembers.scala:515)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.getTypes(TypeDefinitionMembers.scala:517)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.$anonfun$processDeclarations$4(TypeDefinitionMembers.scala:660)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$Lazy.apply(TypeDefinitionMembers.scala:715)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.privateProcessDeclarations(TypeDefinitionMembers.scala:937)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.processDeclarations(TypeDefinitionMembers.scala:662)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.processDeclarationsForTemplateBody(ScTemplateDefinition.scala:393)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.processDeclarationsForTemplateBody$(ScTemplateDefinition.scala:328)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.ScObjectImpl.processDeclarationsForTemplateBody(ScObjectImpl.scala:108)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.templates.ScTemplateBodyImpl.processDeclarations(ScTemplateBodyImpl.scala:77)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.ScStableCodeReferenceElementImpl.treeWalkUp$1(ScStableCodeReferenceElementImpl.scala:307)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.ScStableCodeReferenceElementImpl.processQualifier(ScStableCodeReferenceElementImpl.scala:316)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.ScStableCodeReferenceElementImpl.doResolve(ScStableCodeReferenceElementImpl.scala:482)
	at org.jetbrains.plugins.scala.lang.resolve.StableCodeReferenceElementResolver.resolve(StableCodeReferenceElementResolver.scala:42)
	at org.jetbrains.plugins.scala.lang.resolve.ResolvableStableCodeReferenceElement$Ext$.resolveNoConstructor$extension(ResolvableStableCodeReferenceElement.scala:43)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.innerNonValueType(ScSimpleTypeElementImpl.scala:305)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.innerType(ScSimpleTypeElementImpl.scala:39)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.getType(ScTypeElement.scala:31)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.getType$(ScTypeElement.scala:29)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.getType(ScSimpleTypeElementImpl.scala:38)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.type(ScTypeElement.scala:27)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.type$(ScTypeElement.scala:27)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.type(ScSimpleTypeElementImpl.scala:38)
	at org.jetbrains.plugins.scala.lang.psi.impl.statements.ScFunctionDefinitionImpl.returnTypeInner(ScFunctionDefinitionImpl.scala:79)
	at org.jetbrains.plugins.scala.lang.psi.api.statements.ScFunction$Ext$.returnType$extension(ScFunction.scala:621)
	at org.jetbrains.plugins.scala.lang.psi.light.ScFunctionWrapper.returnType(ScFunctionWrapper.scala:124)
	at org.jetbrains.plugins.scala.lang.psi.light.PsiMethodWrapper.$anonfun$getReturnType$1(PsiMethodWrapper.scala:33)
	at scala.Option.getOrElse(Option.scala:121)
	at org.jetbrains.plugins.scala.lang.psi.light.PsiMethodWrapper.getReturnType(PsiMethodWrapper.scala:32)
	at com.intellij.psi.util.PsiMethodUtil.isMainMethod(PsiMethodUtil.java:59)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.$anonfun$isMainMethod$1(ScalaApplicationConfigurationProducer.scala:167)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.$anonfun$isMainMethod$1$adapted(ScalaApplicationConfigurationProducer.scala:167)
	at scala.Option.exists(Option.scala:240)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.hasJavaMainWrapper$1(ScalaApplicationConfigurationProducer.scala:167)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.isMainMethod(ScalaApplicationConfigurationProducer.scala:169)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$$anonfun$declaredMain$1$1.applyOrElse(ScalaApplicationConfigurationProducer.scala:148)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$$anonfun$declaredMain$1$1.applyOrElse(ScalaApplicationConfigurationProducer.scala:147)
	at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:145)
	at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:132)
	at scala.collection.AbstractTraversable.collectFirst(Traversable.scala:104)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.declaredMain$1(ScalaApplicationConfigurationProducer.scala:147)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.findMainMethodInner$1(ScalaApplicationConfigurationProducer.scala:154)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.findMainMethod(ScalaApplicationConfigurationProducer.scala:158)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.hasMainMethod(ScalaApplicationConfigurationProducer.scala:135)
	at org.jetbrains.plugins.scala.runner.ScalaRunLineMarkerContributor.getInfo(ScalaRunLineMarkerContributor.scala:20)
	at com.google.idea.blaze.scala.run.producers.BlazeScalaRunLineMarkerContributor.getInfo(BlazeScalaRunLineMarkerContributor.java:31)
	at com.intellij.execution.lineMarker.RunLineMarkerProvider.getLineMarkerInfo(RunLineMarkerProvider.java:58)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.a(LineMarkersPass.java:190)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.a(LineMarkersPass.java:112)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:96)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:107)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:437)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1127)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.d(PassExecutorService.java:430)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:534)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:479)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:429)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:405)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:143)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:229)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:403)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:170)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)


java.lang.NoSuchMethodError: scala.Predef$.refArrayOps([Ljava/lang/Object;)Lscala/collection/mutable/ArrayOps;
	at com.spotify.scio.ScioInjector$$anonfun$getConstructorProps$2.apply(ScioInjector.scala:192)
	at com.spotify.scio.ScioInjector$$anonfun$getConstructorProps$2.apply(ScioInjector.scala:193)
	at scala.Option.map(Option.scala:146)
	at com.spotify.scio.ScioInjector.getConstructorProps(ScioInjector.scala:191)
	at com.spotify.scio.ScioInjector.getTupledMethod(ScioInjector.scala:210)
	at com.spotify.scio.ScioInjector$$anonfun$injectInners$1.apply(ScioInjector.scala:110)
	at com.spotify.scio.ScioInjector$$anonfun$injectInners$1.apply(ScioInjector.scala:106)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
	at scala.collection.immutable.List.flatMap(List.scala:352)
	at com.spotify.scio.ScioInjector.injectInners(ScioInjector.scala:106)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.SyntheticMembersInjector$.$anonfun$injectInners$1(SyntheticMembersInjector.scala:132)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.SyntheticMembersInjector$.$anonfun$injectInners$1$adapted(SyntheticMembersInjector.scala:131)
	at scala.collection.immutable.HashSet$HashSet1.foreach(HashSet.scala:320)
	at scala.collection.immutable.HashSet$HashTrieSet.foreach(HashSet.scala:976)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.SyntheticMembersInjector$.injectInners(SyntheticMembersInjector.scala:131)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTypeDefinition.syntheticTypeDefinitionsImpl(ScTypeDefinition.scala:77)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTypeDefinition.syntheticTypeDefinitionsImpl$(ScTypeDefinition.scala:77)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.ScTypeDefinitionImpl.syntheticTypeDefinitionsImpl(ScTypeDefinitionImpl.scala:49)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.syntheticTypeDefinitions(ScTemplateDefinition.scala:183)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.syntheticTypeDefinitions$(ScTemplateDefinition.scala:182)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.ScTypeDefinitionImpl.syntheticTypeDefinitions(ScTypeDefinitionImpl.scala:49)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$TypeNodes$.processScala(TypeDefinitionMembers.scala:282)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.MixinNodes.build(MixinNodes.scala:336)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.MixinNodes.build(MixinNodes.scala:308)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.inner$2(TypeDefinitionMembers.scala:515)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.getTypes(TypeDefinitionMembers.scala:517)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.$anonfun$processDeclarations$4(TypeDefinitionMembers.scala:660)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$Lazy.apply(TypeDefinitionMembers.scala:715)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.privateProcessDeclarations(TypeDefinitionMembers.scala:937)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.TypeDefinitionMembers$.processDeclarations(TypeDefinitionMembers.scala:662)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.processDeclarationsForTemplateBody(ScTemplateDefinition.scala:393)
	at org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.ScTemplateDefinition.processDeclarationsForTemplateBody$(ScTemplateDefinition.scala:328)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.typedef.ScObjectImpl.processDeclarationsForTemplateBody(ScObjectImpl.scala:108)
	at org.jetbrains.plugins.scala.lang.psi.impl.toplevel.templates.ScTemplateBodyImpl.processDeclarations(ScTemplateBodyImpl.scala:77)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.ScStableCodeReferenceElementImpl.treeWalkUp$1(ScStableCodeReferenceElementImpl.scala:307)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.ScStableCodeReferenceElementImpl.processQualifier(ScStableCodeReferenceElementImpl.scala:316)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.ScStableCodeReferenceElementImpl.doResolve(ScStableCodeReferenceElementImpl.scala:482)
	at org.jetbrains.plugins.scala.lang.resolve.StableCodeReferenceElementResolver.resolve(StableCodeReferenceElementResolver.scala:42)
	at org.jetbrains.plugins.scala.lang.resolve.ResolvableStableCodeReferenceElement$Ext$.resolveNoConstructor$extension(ResolvableStableCodeReferenceElement.scala:43)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.innerNonValueType(ScSimpleTypeElementImpl.scala:305)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.innerType(ScSimpleTypeElementImpl.scala:39)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.getType(ScTypeElement.scala:31)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.getType$(ScTypeElement.scala:29)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.getType(ScSimpleTypeElementImpl.scala:38)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.type(ScTypeElement.scala:27)
	at org.jetbrains.plugins.scala.lang.psi.api.base.types.ScTypeElement.type$(ScTypeElement.scala:27)
	at org.jetbrains.plugins.scala.lang.psi.impl.base.types.ScSimpleTypeElementImpl.type(ScSimpleTypeElementImpl.scala:38)
	at org.jetbrains.plugins.scala.lang.psi.impl.statements.ScFunctionDefinitionImpl.returnTypeInner(ScFunctionDefinitionImpl.scala:79)
	at org.jetbrains.plugins.scala.lang.psi.api.statements.ScFunction$Ext$.returnType$extension(ScFunction.scala:621)
	at org.jetbrains.plugins.scala.lang.psi.light.ScFunctionWrapper.returnType(ScFunctionWrapper.scala:124)
	at org.jetbrains.plugins.scala.lang.psi.light.PsiMethodWrapper.$anonfun$getReturnType$1(PsiMethodWrapper.scala:33)
	at scala.Option.getOrElse(Option.scala:121)
	at org.jetbrains.plugins.scala.lang.psi.light.PsiMethodWrapper.getReturnType(PsiMethodWrapper.scala:32)
	at com.intellij.psi.util.PsiMethodUtil.isMainMethod(PsiMethodUtil.java:59)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.$anonfun$isMainMethod$1(ScalaApplicationConfigurationProducer.scala:167)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.$anonfun$isMainMethod$1$adapted(ScalaApplicationConfigurationProducer.scala:167)
	at scala.Option.exists(Option.scala:240)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.hasJavaMainWrapper$1(ScalaApplicationConfigurationProducer.scala:167)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.isMainMethod(ScalaApplicationConfigurationProducer.scala:169)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$$anonfun$declaredMain$1$1.applyOrElse(ScalaApplicationConfigurationProducer.scala:148)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$$anonfun$declaredMain$1$1.applyOrElse(ScalaApplicationConfigurationProducer.scala:147)
	at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:145)
	at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:132)
	at scala.collection.AbstractTraversable.collectFirst(Traversable.scala:104)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.declaredMain$1(ScalaApplicationConfigurationProducer.scala:147)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.findMainMethodInner$1(ScalaApplicationConfigurationProducer.scala:154)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.findMainMethod(ScalaApplicationConfigurationProducer.scala:158)
	at org.jetbrains.plugins.scala.runner.ScalaMainMethodUtil$.hasMainMethod(ScalaApplicationConfigurationProducer.scala:135)
	at org.jetbrains.plugins.scala.runner.ScalaRunLineMarkerContributor.getInfo(ScalaRunLineMarkerContributor.scala:20)
	at com.google.idea.blaze.scala.run.producers.BlazeScalaRunLineMarkerContributor.getInfo(BlazeScalaRunLineMarkerContributor.java:31)
	at com.intellij.execution.lineMarker.RunLineMarkerProvider.getLineMarkerInfo(RunLineMarkerProvider.java:58)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.a(LineMarkersPass.java:190)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.a(LineMarkersPass.java:112)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:96)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:107)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:437)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1127)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.d(PassExecutorService.java:430)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:534)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:479)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:429)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:405)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:143)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:229)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:403)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:170)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

**Comment by nevillelyh** [comment]
Reported as https://github.com/spotify/scio-idea-plugin/issues/12
Closing this.

**Comment by mikaelstaldal** [comment]
OK. I was not aware of that the plugin had a separate repository. The info page for the plugin links here, maybe it should link to the other repository?

**Comment by ravwojdyla** [comment]
@mikaelstaldal fixed in just released 0.1.10. Thanks for reporting.
